apisix/timers.lua

```diff
@@ -0,0 +1,90 @@
+--
+-- Licensed to the Apache Software Foundation (ASF) under one or more
+-- contributor license agreements.  See the NOTICE file distributed with
+-- this work for additional information regarding copyright ownership.
+-- The ASF licenses this file to You under the Apache License, Version 2.0
+-- (the "License"); you may not use this file except in compliance with
+-- the License.  You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+--
+local core = require("apisix.core")
+local process = require("ngx.process")
+local pairs = pairs
+local unpack = unpack
+local thread_spawn = ngx.thread.spawn
+local thread_wait = ngx.thread.wait
+
+
+local timers = {}
+
+
+local _M = {
+}
```

Comment:
I think this `local _M = {}` format will look better, what do you think?

Reply:
@Firstsawyou 
Updated in the new commit.